content/en/tracing/profiling/getting_started.md

```diff
@@ -0,0 +1,209 @@
+
+2. Get `dd-trace-go` using the command:
+
+    ```shell
+    go get gopkg.in/DataDog/dd-trace-go.v1
+    ```
+
+     **Note**: Profiling is available in the `dd-trace-go` library for versions 1.23.0+.
+
+3. Import the [profiler][2] at the start of your application:
+
+    ```Go
+    import "gopkg.in/DataDog/dd-trace-go.v1/profiler"
+    ```
+
+4. Add the following snippet to start the profiler:
+
+    ```Go
+    err := profiler.Start(
+        profiler.WithService("<SERVICE_NAME>"),
+        profiler.WithEnv("<ENVIRONMENT>"),
+        profiler.WithVersion("<APPLICATION_VERSION>"),
+        profiler.WithTags("<KEY1>:<VALUE1>, <KEY2>:<VALUE2>"),
+    )
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer profiler.Stop()
+    ```
+
+4. After a minute or two, visualize your profiles in the [Datadog APM > Profiling page][3].
+
+**Note**:
+
+- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use profiler configuration to set the parameters:
+
+| Method | Type          | Description                                                                                                  |
+| ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------ |
+|  WithService     | String        | The Datadog [service][4] name, for example `my-web-app`.             |
+|  WithEnv         | String        | The Datadog [environment][5] name, for example `production`.         |
+|  WithVersion     | String        | The version of your application.                                                                             |  
+|  WithTags        | String        | The tags to apply to an uploaded profile. Must be a list of in the format `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`. |
+
+- Alternatively you can also set the profiler configuration using environment variables:
+
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_SERVICE`                                     | String        | The Datadog [service][4] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][5] name, for example `production`. |
+| `DD_VERSION`                                     | String        | The version of your application.                             |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.   |
```

Comment:
Here too :-)

Reply:
It seems verbose to duplicate the types and descriptions rather than keeping both config mechanisms in a single table. Up to you.